compute_endpoint/globus_compute_endpoint/logging_config.py

```diff
@@ -66,14 +75,12 @@ class ComputeConsoleFormatter(logging.Formatter):
     uuid_re = re.compile(rf"(?<!\dm)({_uuid_re})")
 
     def __init__(
-        self,
-        debug: bool = False,
-        no_color: bool = False,
-        fmt: str = "",
-        datefmt: str = "%Y-%m-%d %H:%M:%S",
+        self, debug: bool = False, no_color: bool = False, fmt: str = "", **kwargs
     ) -> None:
         super().__init__()
 
+        kwargs.setdefault("datefmt", "%Y-%m-%d %H:%M:%S,%f")
```

Comment:
I don't like the milliseconds field in the logs. At the same time, it does help with debug logging. Do you think it would work if we have different format strings based on log level?